Description: Install in /usr/lib instead of /usr/lib64, even on 64-bit archs
Origin: Debian
---
 CMakeLists.txt |    7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

--- libdrumstick.orig/CMakeLists.txt
+++ libdrumstick/CMakeLists.txt
@@ -27,11 +27,8 @@ IF(COMMAND CMAKE_POLICY)
     CMAKE_POLICY(SET CMP0003 NEW)
 ENDIF(COMMAND CMAKE_POLICY)
 
-IF(CMAKE_SIZEOF_VOID_P MATCHES "8")
-    SET(_INIT_LIB_SUFFIX "64")
-ELSE(CMAKE_SIZEOF_VOID_P MATCHES "8")
-    SET(_INIT_LIB_SUFFIX "")
-ENDIF(CMAKE_SIZEOF_VOID_P MATCHES "8")
+# Debian, unlike the FHS, does not use a '64' suffix like 'lib64'
+SET(_INIT_LIB_SUFFIX "")
 SET(LIB_SUFFIX ${_INIT_LIB_SUFFIX} CACHE STRING "optional library install directory suffix: 32, 64, none")
 
 #Comment out if relative paths break the link scripts
